easybuild/easyconfigs/s/SWIG/SWIG-4.0.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,21 @@
+name = 'SWIG'
+version = '4.0.0'
```

Comment:
There is a SWIG 3.0.12 at GCCcore/8.2.0

If we need a this newer version (which I suppose is OK, as it's just build-dep), then it should be at GCCcore level from now.